<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionProgressDialog</class>
 <widget class="QMainWindow" name="ConversionProgressDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConversionProgressDialog</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Progress</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="conversionProgressLabel">
         <property name="text">
          <string>Conversion Progress:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="conversionProgressBar"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="transformationProgressLabel">
         <property name="text">
          <string>Transformation Progress:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="transformationProgressBar"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="cachingProgressLabel">
         <property name="text">
          <string>Caching Progress:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QProgressBar" name="cachingProgressBar"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="savingProgressLabel">
         <property name="text">
          <string>Saving Progress:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QProgressBar" name="savingProgressBar"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
    <item>
     <widget class="QTreeView" name="progressTreeView">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="logListWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
